### DISCUSSION TITLE
How to analyze accessing a static property on a class?

### DISCUSSION THREAD

**Discussion by dmrickey** [invertase/dart_custom_lint#165] [Q&A] [answered]
I'm trying to write a rule that suggests to not use `Alignment` (so if it sees something like `Alignment.bottomRight` it will show a warning that isn't RTL compatible. But I can't find an analyzer that seems appropriate for it. `addExpression` and `addIdentifier` both find it, but they also find waaay more so it seems like there should be something more specific, but I'm not finding it. Am I missing something obvious?

**Comment by rrousselGit** [comment, marked as answer]
`addIdentifier` is what you want. `bottomRight` is an identifier
